packages/dataviews/src/style.scss

```diff
@@ -287,7 +287,23 @@
 	padding: 0 $grid-unit-40;
 }
 
+.dataviews-table-selection-checkbox label {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
+	white-space: nowrap;
+	border: 0;
+}
+
 .dataviews__filters-custom-menu-radio-item-prefix {
 	display: block;
 	width: 24px;
 }
+
+.dataviews-bulk-edit-button.components-button.is-secondary {
+	padding-right: $grid-unit-15 + 2px;
```

Comment:
Why we need a special padding in the first place. For me, the issue seems to be that the flex behavior is "shrinking" the width of the button so instead of a custom padding (which result in a wrong balance for me btw), we need to use `flex-string: 0`

Reply:
Good one flex-shrink did the trick. I did not find the root cause of why the button was cut on the right and the padding was like a last resource to address the issue.